tests/drivers/uart/uart_emul/src/main.c

```diff
@@ -9,25 +9,30 @@
 #include <zephyr/drivers/serial/uart_emul.h>
 #include <zephyr/ztest.h>
 
-#define EMUL_UART_NODE	       DT_NODELABEL(euart0)
+#define EMUL_UART_NODE         DT_NODELABEL(euart0)
```

Comment:
Please remove the whitespace change. This is not related.

Reply:
oops, i fixed that